src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -22,4 +22,8 @@ public User get(String login) {
         }
         return null;
     }
+
+    @Override
+    public void deleteAll() {
```

Comment:
where is the implementation of #deleteAll? 
unused\not implemented methods should be removed.